dags/veda_data_pipeline/veda_collection_pipeline.py

```diff
@@ -14,12 +14,12 @@
     "schedule_interval": None,
     "catchup": False,
     "doc_md": dag_doc_md,
-    "tags": ["collection", "discovery"],
+    "tags": ["collection"],
 }
 
-with DAG("veda_dataset_pipeline", **dag_args) as dag:
+with DAG("veda_collection_pipeline", **dag_args) as dag:
     start = EmptyOperator(task_id="start", dag=dag)
-    end = EmptyOperator(task_id="end", trigger_rule=TriggerRule.ONE_SUCCESS, dag=dag)
+    end = EmptyOperator(task_id="end", trigger_rule=TriggerRule.NONE_FAILED_MIN_ONE_SUCCESS, dag=dag)
```

Comment:
This last task is a `no-op`, and keeps getting skipped. I'm not sure why. The DAG still gets marked as a success or failure as appropriate, but I think there's a trigger rule nuance I'm missing.

Reply:
I think NONE_FAILED be enough? there were only two direct upstream tasks, right? Should handle the skipped one as well - unless, is there a scenario where both direct upstreams are skipped?

Reply:
I haven't tried it myself yet, but @amarouane-ABDELHAK thinks `ONE_SUCCESS` is the correct trigger here.

Reply:
None_failed can work too, suggested ONE_SUCCESS because we  expect at any giving time only one task to be successful